GVFS/GVFS.Platform.Mac/ProjFSKext.cs

```diff
@@ -44,16 +50,21 @@ public bool IsSupported(string normalizedEnlistmentRootPath, out string warning,
 
         public string FlushLogs()
         {
-            throw new NotImplementedException();
+            Directory.CreateDirectory(this.LogsFolderPath);
+            ProcessResult logShowOutput = ProcessHelper.Run("log", args: "show --predicate \"subsystem contains \'org.vfsforgit\'\" --info", redirectOutput: true);
+            File.WriteAllText(Path.Combine(this.LogsFolderPath, "PrjFSKext.log"), logShowOutput.Output);

```

Comment:
This might be a lot of text to store in memory.  Can we stream the log from stdOut to the log file?

Reply:
@nickgra could you file a follow-up work item to stop using `File.WriteAllText`?

Reply:
#743